apps/api/src/server/applications/application/application.service.js

```diff
@@ -95,12 +95,16 @@ export const startApplication = async (id) => {
 
 	// sendMessage({test: 'test'}, 'test')
 
-	await caseRepository.updateApplicationStatusAndDataById(caseDetails.id, {
-		status: nextStatusForRepository,
-		data: {},
-		currentStatuses: caseDetails.CaseStatus,
-		setReference: true
-	});
+	await caseRepository.updateApplicationStatusAndDataById(
+		caseDetails.id,
+		{
+			status: nextStatusForRepository,
+			data: {},
+			currentStatuses: caseDetails.CaseStatus,
+			setReference: true
+		},
+		[folderRepository.createFolders()]
```

Comment:
We need to add the case ID to the arguments of this function, right?

Reply:
yes i thought i did hmm

Reply:
This is still missing

Reply:
it's showing up as there on my github - has it maybe updated?